### DISCUSSION TITLE
FreeBSD run python script as Daemon

### DISCUSSION THREAD

**Discussion by WienerKinderSchnitzel** [curzon01/mqtt2mysql#20] [Q&A] [answered]
Hy,
i just run this on freeBSD and just started the script by screen with some littele modifications. 
I noticed some issue while creating the triggers on mysql 5.7 also the called python3 will be python39 on freeBSD

May its possible to run as daemon? 
I haven't try this now.
Mike

**Comment by curzon01** [comment, marked as answer]
5.7 reaches EOL this year, so no adaption planned :wink:, great it' works for you

systemd is not an option for [running as a daemon](https://github.com/curzon01/mqtt2sql#start-as-systemd-manager-daemon)?
